components/normalizer/benches/data/udhr_vie.txt

```diff
@@ -0,0 +1,221 @@
+Universal Declaration of Human Rights - Vietnamese
```

Comment:
Need to investigate this. CC @skywalkerswartz

Reply:
Looks like you updated it ! thanks for checking all these licence headers !